### ISSUE TITLE
Tinymce >= 5.1.0 not supported

### ISSUE THREAD

**Issue by jose-barata** [tinymce/tinymce-angular#117]
**What is the current behavior?**
None of the menus or buttons is responsive.

**To reproduce:**
* Create a new angular project
* Install @tinymce/tinymce-angular: 3.3.1 and tinymce: 5.1.1
* Follow the steps to use tinymce locally

**Which versions of TinyMCE/TinyMCE-Angular, and which browser / OS are affected by this issue? 
**Browsers tested:**
* Chrome latest (@07/11/2019)
* Firefox latest (@07/11/2019)

**Dependencies installed:**
@tinymce/tinymce-angular: 3.3.1
tinymce: 5.1.1

Did this work in previous versions of TinyMCE or TinyMCE-Angular?** Yes, in 5.0.16

**Comment by SimonFc** [comment]
Hi @jose-barata  I set the cloudChannel to 5-dev on this [CodeSandbox](https://codesandbox.io/s/tinymceangular-wyje4?fontsize=14) example. I'm unable to replicate your issue, is there perhaps something else you did which might have caused it? (I've tried locally as well)

**Comment by jose-barata** [comment]
@SimonFc Thanks for getting back to me so quickly. So I'm in a funny situation and I'm going to try to provide as much detail as I can. I'm working with another colleague. He is working on Linux. We have the same test project and we are both with 5.1.1. First we found that he didn't encounter any problems, using the project locally or pointing to my machine, on both Chrome and Firefox (both on the lastest versions). I'm using Windows 10. We have other colleagues with Windows 10 and we had them pointing to my machine as well, again no issues at all. I've tried Chrome and Firefox without any success but it on IE11 it worked brilliantly (go figure it out that one). I have to tell you that I use a dual monitor setup...two monitors connected to a laptop (and laptop lid closed). I've tried to use only one monitor connected to laptop and I was amazed when all functionality works as expected.

I can also tell you that I have no problems with version 5.0.16 but I face this issues from 5.1.0.

**Comment by SimonFc** [comment]
So I'm assuming that it's not working for you on FF/Chrome using Windows? I'm still unable to replicate the issue you're describing. Are you sure that there's nothing else(code-wise) that you did you encounter this issue? See if you can modify the CodeSandbox example. (If we can't replicate it there's not much we can do)

**Comment by jose-barata** [comment]
Not working in FF/Chrome using Windows 10 with two monitors plugged in into a laptop using a hdmi to usb converter (if that matters). Everything works on IE11. The code is absolutely the same between fully working and not working by only upgrading the tinymce version from 5.016 to 5.1.1.

I can also tell you that with version 5.1.1 if I have my setup the way it is, the editor will not be displayed/work as expected. However, if I connect only one monitor directly to the laptop the editor will work as expected (zero code changes). Funny right? Also...with version 5.0.16 it works no matter what.

Probably I should open an issue here https://github.com/tinymce/tinymce/issues instead ?

**Comment by SimonFc** [comment]
It's alright if we keep the issue here. And lets assume that the monitor setup you have is irrelevant for now unless this is a browser bug.

We need actual code to be able to test this. I'm sure there's something more to this than just "version 5.1.1". If you can't replicate this on CodeSandbox, can you perhaps set up a small repo showcasing the issue?

**Comment by jose-barata** [comment]
You are absolutely right!

Here is the test repo:
https://github.com/jose-barata/tinymce-test

I've recorded the problem here:
https://webm.red/view/XGlJ.webm

**Comment by SimonFc** [comment]
Thank you for setting up the repo, it's appreciated. I'm still unable to reproduce the issue though. Tested on Chrome 78.0.3904.97 and Firefox 70.0.1. The only thing I can think of now is that perhaps you have some extension that's interfering? Perhaps far fetched since I'm assuming that you would be using it on both your Chrome and Firefox browsers.

Is it working for you [here](http://fiddle.tinymce.com/YUgaab?

**Comment by jose-barata** [comment]
No, it's not working in that fiddle. Also thought that about the extensions so I disabled them all and restarted the browsers (Chrome 78.0.3904.97 and FF 70.0.1) - the issue persists. As a side note, it works perfectly on IE11.

With the fiddle that you've created I can tell you that it doesn't work when I'm connected with my laptop to the two monitors (setup that I've previously mentioned) but If I disconnect them and use the laptop screen...it works.

I also went to the length of installing a linux distro in a VM and accessing your fiddle with the same dual monitor setup and ... it works.

I'm pretty sure that will be super hard to reproduce unless you have the same hardware and OS combo.

**Comment by jose-barata** [comment]
> No, it's not working in that fiddle. Also thought that about the extensions so I disabled them all and restarted the browsers (Chrome 78.0.3904.97 and FF 70.0.1) - the issue persists. As a side note, it works perfectly on IE11.
> 
> With the fiddle that you've created I can tell you that it doesn't work when I'm connected with my laptop to the two monitors (setup that I've previously mentioned) but If I disconnect them and use the laptop screen...it works.
> 
> I also went to the length of installing a linux distro in a VM and accessing your fiddle with the same dual monitor setup and ... it works.
> 
> I'm pretty sure that will be super hard to reproduce unless you have the same hardware and OS combo.

Forgot to mention that I always need to restart the browser after I connect/disconnect the monitors to make it work/not work

**Comment by SimonFc** [comment]
Are any of the monitors you're using a touch device? If so, I'll be guessing that it works alright if you connect the monitor(s) after the fact that TinyMCE has initialized?

**Comment by angeluu** [comment]
Did you guys got this working .. I am facing the same issue now.
It was working perfectly in the Angular 4 version, Yesterday I upgraded it to Angular 8 and it stopped working. It's not showing any errors, just the editor is not showing. Any idea how to fix this.
These are my dependencies:
 "dependencies": {
"@angular/core": "^8.2.13",
"@tinymce/tinymce-angular": "^3.3.1",
"tinymce": "^5.1.1",
}

**Comment by SimonFc** [comment]
@a4DEVP I don't think this is the same issue since since the editor doesn't show at all for you. I'm unable to reproduce the issue you have with the versions of the packages you have specified so I'm guessing the source of your problem is something other than the packages simply not being compatible. Open up a new issue with detailed steps for replicating it and we can take it from there. Thanks!

**Comment by a4DEVP** [comment]
Hi SimonFc, Thank u for the response.
Not sure what was the real cause of the issue. But I reverted the TinyMce plugin to the previous version ( "@tinymce/tinymce-angular": "^1.0.0", ) and now it working.

**Comment by jose-barata** [comment]
> Are any of the monitors you're using a touch device? If so, I'm guessing that it works alright if you connect the monitor(s) after the fact that TinyMCE has initialized?

Yes to both questions. Sorry it took me so long to reply but I was away.

**Comment by SimonFc** [comment]
No worries! I know what the issue is now - it's one related to device detection in TinyMCE which is done when TinyMCE initializes, and depending on the device type it'll handle touch & click events differently. I'll move this issue to the `tinymce` since it's not due to the `tinymce-angular` wrapper
